dali/sasha/saxref.cpp

```diff
@@ -1228,6 +1230,13 @@ class CNewXRefManager: public CNewXRefManagerBase
             isPlaneStriped = !storagePlane->hasProp("@hostGroup") && (numDevices>1);
             numStripedDevices = isPlaneStriped ? numDevices : 1;
         }
+
+        if (!isEmptyString(_scopeFilters))
+        {
+            log(false, "Filter Scopes Enabled: searching for files in: %s", _scopeFilters);
+            filterScopesEnabled = true;
+            scopeFilters.appendList(_scopeFilters, "::");
```

Comment:
just to check (it looks like it) but to verify. This is breaking down a single scope filter (e.g. thor400::jakefiles::banking) into the a scope string it will filter on as it descends.
(worth adding a comment in code if not elsewhere).

So I think that means, you couldn't currently filter by >1 scope, e.g. thor400::jakesfiles::banking, thor400::jakesfiles::insurane

That's fine, may never be needed, just checking.